src/main/java/io/github/mountainrange/mule/managers/KeyBindPackage.java

```diff
@@ -0,0 +1,21 @@
+package io.github.mountainrange.mule.managers;
+
+import io.github.mountainrange.mule.GameManager;
+import io.github.mountainrange.mule.gameplay.WorldMap;
+
+/**
+ * A wrapper class that gives KeyFunctions information/access to the state of the game.
+ */
+public class KeyBindPackage {
+	/*
+	 * We don't care too much about getters and setters here, since this is used
+	 * only once...
+	 */
+	public GameManager manager;
+	public WorldMap map;
+
+	public KeyBindPackage(GameManager m, WorldMap map) {
+		this.manager = m;
+		this.map = map;
+	}
+}
```

Comment:
I would personally name this `GameState`, since we're passing information about the game state to the `KeyFunction`, and name `GameState` something else (like `GameView` or something, since it's just what differentiates different keybindings), but that's just me.